lib/map_elevation.dart

```diff
@@ -18,35 +18,40 @@ class Elevation extends StatefulWidget {
 
   /// Elevation gradient colors
   /// See [ElevationGradientColors] for more details
-  final ElevationGradientColors elevationGradientColors;
+  final ElevationGradientColors? elevationGradientColors;
 
   /// [WidgetBuilder] like Function to add child over the graph
-  final Function(BuildContext context, Size size) child;
+  final Function(BuildContext context, Size size)? child;
 
   Elevation(this.points,
-      {this.color, this.elevationGradientColors, this.child});
+      {required this.color, this.elevationGradientColors, this.child});
 
   @override
   State<StatefulWidget> createState() => _ElevationState();
 }
 
 class _ElevationState extends State<Elevation> {
-  double _hoverLinePosition;
-  double _hoveredAltitude;
+  double? _hoverLinePosition;
+  double? _hoveredAltitude;
 
   @override
   Widget build(BuildContext context) {
     return LayoutBuilder(builder: (BuildContext context, BoxConstraints bc) {
       Offset _lbPadding = Offset(35, 6);
       _ElevationPainter elevationPainter = _ElevationPainter(widget.points,
           paintColor: widget.color,
-          elevationGradientColors: widget.elevationGradientColors,
+          elevationGradientColors: widget.elevationGradientColors ??
+              ElevationGradientColors(
+                  gt10: Colors.green,
+                  gt20: Colors.orangeAccent,
+                  gt30: Colors.redAccent),
```

Comment:
This parameter should not be defaulted to anything to allow disabling elevation gradients.

See:  
https://github.com/OwnWeb/map_elevation/blob/b37c46d34699efc394d1f5c11c3b0de7d9a58501/lib/map_elevation.dart#L152-L168

Reply:
Right, will do it